src/components/molecules/UserProfilePicture/UserProfilePicture.styles.ts

```diff
@@ -35,36 +38,14 @@ export const Reaction = styled.div<ReactionProps>`
     reactionPosition === "right" ? reactionRight : reactionLeft};
   top: -25px;
   // @debt convert this to scss then use our z-index layer helper here
-  z-index: 10;
+  z-index: ${reactionZIndex};
 
   font-size: 50px;
 
   animation: pulse 4s ease-in-out infinite;
 `;
 
-// --- Reaction Container
-export const Container = styled.div`
-  height: 100%;
-  position: relative;
-
-  background-position: center;
-  background-size: cover;
-  // @debt convert this to scss then use our z-index layer helper here
-  z-index: 11;
-
-  ${Avatar} {
-    border-radius: 10rem;
-  }
-
-  .on-right {
-    ${Reaction} {
-      right: unset;
-      left: -20px;
-    }
-  }
-`;
```

Comment:
Why was this moved? Presumably it worked fine where it was. Adds extra churn to this diff for little to no reason IMO.